### DISCUSSION TITLE
MONAI nodule detection model training and handling  zero nodule datasets

### DISCUSSION THREAD

**Discussion by DSRajesh** [Project-MONAI/MONAI#8119] [Q&A]
Hello
Thank you for the MONAI team.  I have been training a nodule detection model using 
![image](https://github.com/user-attachments/assets/f7359183-358b-45ae-8627-e0bfb3e54706)

Among my datasets I have some which have 0 nodules in them.  And i think i get a error log as shown below for such cases

![image](https://github.com/user-attachments/assets/c86af7d4-df66-4676-aa5d-6b77f1b21671)
I have applied the correction mentioned there 

i.e 
boxes = torch.reshape ( boxes, [ 0, 4 ] ) as follows:

![image](https://github.com/user-attachments/assets/d1278e9e-ae37-4052-82f6-cc22e18a2812)

but i still get the same error log as shown above.  May i know the reason for this and the solution for the same please.  Also I have used datasets that have 0 nodules and a nnDetection (https://github.com/MIC-DKFZ/nnDetection) model trains well under such situations.

Rajesh D S

**Comment by KumoLiu** [comment]
Hi @DSRajesh, could you please refer whether the discussion [here](https://github.com/Project-MONAI/tutorials/discussions/1197) helps you? Thanks.

**Reply by DSRajesh** [reply]
Hello @KumoLiu 
Thank you for the response.  The discussion you pointed out me to in your response discusses the same solution that i had already tried  as follows:

 
![image](https://github.com/user-attachments/assets/8c9f4805-472d-4aed-b95a-0dea1c6a7794)

and had still resulted in error.

Also the discussion (https://github.com/Project-MONAI/tutorials/discussions/1197) was with reference to MONAI tutorials for "object detection" (https://github.com/Project-MONAI/tutorials/tree/main/detection).   I am using the MONAI zoo model for nodule detection i.e 

![Screenshot from 2024-09-30 13-59-04](https://github.com/user-attachments/assets/cffd7b94-68fc-4100-bd86-6dc7d4cb7564).

Do I need to add some lines in the config/train.json file to incorporate the corrections mentioned in https://github.com/Project-MONAI/tutorials/discussions/1197 ?

Thank you
Rajesh D S
